zgw_consumers/models/services.py

```diff
@@ -193,7 +204,7 @@ def get_service(cls, url: str) -> Optional["Service"]:
         return None
 
     @classmethod
-    def get_client(cls, url: str, **kwargs) -> Optional[ZGWClient]:
+    def get_client(cls, url: str, **kwargs) -> Optional["ZGWClient"]:
```

Comment:
you can use `from __future__ import annotations` so you don't have to quote the thing, and then you can do:

```suggestion
    def get_client(cls, url: str, **kwargs) -> ZGWClient | None:
```